.github/workflows/docker_build.yml

```diff
@@ -0,0 +1,75 @@
+on:
```

Comment:
If I'm reading this right then it looks like this is just for the lambda images we build - could we have something in the file name to reflect this? May be confused with building app images.